Tests/SmokeTestWin32Console/README.md

```diff
@@ -0,0 +1,9 @@
+# About SmokeTestWin32Console
+
+This is a very simple program showing a Realm instance being loaded, data written and queried, in a pure Console app.
+
+It builds for x86 and x64.
+
+Note that it only uses the standalone Realm.Database nuget.
+
+The project was created and tested under Visual Studio 2017 RC and may run into minor project issues with earlier versions.
```

Comment:
I don't know but I had some weirdness when I tried going back to VS2015 - IDE crashed.